codec-compression/src/test/java/io/netty/handler/codec/compression/ZstdEncoderTest.java

```diff
@@ -37,12 +37,21 @@ public class ZstdEncoderTest extends AbstractEncoderTest {
     @Mock
     private ChannelHandlerContext ctx;
 
+    protected static final byte[] BYTES_HUGE = new byte[256 * 1024 * 1024];
```

Comment:
I think it would be better to just allocate this in `hugeData` so it can be GC'ed once the test completes.